component/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+witx_bindgen_rust::export!("../spec/component.witx");
+
+struct Component;
+
+impl component::Component for Component {
+    fn consume_add(lh: i32, rh: i32) -> i32 {
+        lh + rh
```

Comment:
this is awesome, but we have lost the round trip back to the host (showing 2-way interface type calling)

np I can add it back in